doc/misc/npm-faq.md

```diff
@@ -223,7 +207,7 @@ an argument to `require()`.
 
 ## `"node_modules"` is the name of my deity's arch-rival, and a Forbidden Word in my religion.  Can I configure npm to use a different folder?
 
-No.  This will never happen.  This question comes up sometimes,
+No, unfortunately.  This question comes up sometimes,
```

Comment:
See above. It is important that this aspect of npm's design be unambiguous and clear, and this is not something for which the team feels any regret. This is a heavily premeditated, important design choice, and one that has been the target of considerable bikeshedding and attempts at relegislation.